common/src/main/java/com/wynntils/models/players/type/GuildRank.java

```diff
@@ -13,12 +13,26 @@ public enum GuildRank {
     OWNER("Owner");
 
     private final String name;
+    private final String stars;
 
     GuildRank(String name) {
         this.name = name;
+
+        switch (this.name) {
+            case "Owner" -> stars = "★★★★★";
+            case "Chief" -> stars = "★★★★";
```

Comment:
Actually, one last thing - can we remove this switch and just have it directly in the enum values?